packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -101,6 +101,107 @@ enum HeroFlightDirection {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=Be9UH1kXFDw}
 ///
+/// {@tool dartpad --template=stateful_widget_scaffold_center}
+/// This sample shows a `Hero` with List item along with Icon: here we will be using a
+/// rectangle widget for icon. Icon widget in list item at first page , with Hero
+/// widget in list and Enlarged icon to display on Second Page.
```

Comment:
This is a little confusing, `Icon` typically means the `Icon` widget, but in this sample you are using a Container with a color.

```suggestion
/// {@tool dartpad --template=stateful_widget_scaffold_center}
/// This sample displays shows a `Hero` that flies between the two screens
/// and grows to fit the layout on the second page.
```